#!/usr/bin/python3

print('FAIL')
exit(1)
